micrometer-commons/src/main/java/io/micrometer/common/docs/KeyName.java

```diff
@@ -46,6 +46,14 @@ default KeyValue withValue(String value) {
         return KeyValue.of(this, value);
     }
 
+    /**
+     * Creates a key value for the given key name with {@value KeyValue#NONE_VALUE}.
+     * @return key value
+     */
```

Comment:
Could you please add `@since 1.16.0`?

Reply:
@jonatan-ivanov Sure, I added it in d633e82.